.github/workflows/integration_test.yaml

```diff
@@ -12,4 +12,4 @@ jobs:
       juju-channel: 3.3/stable
       self-hosted-runner: true
       charmcraft-channel: latest/edge
-      modules: '["test_action.py", "test_config.py", "test_charm.py", "test_http_interface.py", "test_cos.py"]'
+      modules: '["test_action.py", "test_config.py", "test_charm.py", "test_http_interface.py", "test_cos.py", "test_ingress.py"]'
```

Comment:
In https://github.com/canonical/haproxy-operator/pull/40 I removed this line so will run all the integration tests in the directory